### ISSUE TITLE
TypeError: Cannot redefine property: name

### ISSUE THREAD

**Issue by ksullivan** [rapid7/le_node#126]
How to replicate:

node version v0.12.15
npm version 3.10.6

Commands:

mkdir le_node_winston_bug
cd le_node_winston_bug
npm install le_node@1.6.2 winston@2.2.0
node --eval "require('le_node')"

/home/kevin/Documents/le_node_winston_bug/node_modules/core-js/library/fn/object/define-property.js:4
  return $Object.defineProperty(it, key, desc);
                 ^
TypeError: Cannot redefine property: name
    at Object.defineProperty (native)
    at defineProperty (/home/kevin/Documents/le_node_winston_bug/node_modules/core-js/library/fn/object/define-property.js:4:18)
    at defineProperties (/home/kevin/Documents/le_node_winston_bug/node_modules/babel-runtime/helpers/createClass.js:18:36)
    at /home/kevin/Documents/le_node_winston_bug/node_modules/babel-runtime/helpers/createClass.js:24:22
    at /home/kevin/Documents/le_node_winston_bug/node_modules/le_node/lib/logger.js:796:35
    at Function.provisionWinston (/home/kevin/Documents/le_node_winston_bug/node_modules/le_node/lib/logger.js:877:8)
    at Object.<anonymous> (/home/kevin/Documents/le_node_winston_bug/node_modules/le_node/lib/logger.js:905:21)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)

**Comment by ksullivan** [comment]
It looks like this is a regression from 1.6.0 to 1.6.2.

**Comment by stopal-r7** [comment]
Cannot reproduce this with node 6.2.0, npm 3.10.3, trying to find a way to test with your node version. No change was applied to winston - logger support in recent fix though.

**Comment by stopal-r7** [comment]
Hi @ksullivan, I created a PR for this. 

Recently we updated our travis nodejs build versions to 4, 5 and 6 and added eslint integration. ES6 is not supported on node 0.12.x, that's why eslint and tests are failing as they are running against the actual es6 code. For that reason, we cannot enable 0.12.x builds on travis. This PR should fix your build problems as far as I have tested by temporarily disabling eslint for 0.12.x and used --harmony option for tests.

Thanks for the feedback!
